Engine/Source/ModuleEditor.h

```diff
@@ -30,6 +34,10 @@ class ModuleEditor : public Module
 	bool mDrawRaycast = false;
 	std::map<const char*, Panel*> mPanels;
 	void OpenLoadScene();
+
+	//ImGuizmo variables
+	ImGuizmo::OPERATION mCurrentGizmoOperation = ImGuizmo::ROTATE;
```

Comment:
In the ModuleScene, the `OPERATIONis `initialized as `TRANSLATE`, and the `MODE `as `LOCAL`. Should be the same to mantain consistency?

Reply:
Actually these variables are not used there, I will delete them. I think I forgot them there when I was deciding where to implement the guizmo feature.